ini/stm32f4.ini

```diff
@@ -67,11 +67,9 @@ upload_protocol     = dfu
 #
 [env:FYSETC_S6]
 platform                    = ${common_stm32.platform}
-extends                     = stm32_variant
-board                       = marlin_fysetc_s6
-board_build.offset          = 0x10000
-board_upload.offset_address = 0x08010000
-build_flags                 = ${stm32_variant.build_flags} -DHAL_PCD_MODULE_ENABLED
+extends                     = common_stm32
+board                       = fysetc_s6
```

Comment:
With this changes the board stoped working to me. please try:
`#
# FYSETC S6 (STM32F446RET6 ARM Cortex-M4)
#
[env:FYSETC_S6]
platform                    = ${common_stm32.platform}
extends                     = stm32_variant
board                       = marlin_fysetc_s6
board_build.offset          = 0x10000
board_upload.offset_address = 0x08010000
build_flags                 = ${stm32_variant.build_flags} -DHAL_PCD_MODULE_ENABLED
debug_tool                  = stlink
upload_protocol             = dfu
upload_command              = dfu-util -a 0 -s 0x08010000:leave -D "$SOURCE"

#
# FYSETC S6 new bootloader
#
[env:FYSETC_S6_8000]
platform                    = ${common_stm32.platform}
extends                     = env:FYSETC_S6
board                       = marlin_fysetc_s6
board_build.offset          = 0x8000
board_upload.offset_address = 0x08008000
upload_command              = dfu-util -a 0 -s 0x08008000:leave -D "$SOURCE"`

Reply:
With all the commits, it works in my side. Just keep in mind that you need respective bootloader. If you use FYSETC_S6_8000 env, you need `Bootloader-FYSETC_S6.hex`, `FYSETC_S6` env you need `Bootloader-FYSETC_S6_10000.hex`. You can find the bootloader [here](url).